docs/vanilla.md

```diff
@@ -0,0 +1,144 @@
+* vSphere - Version 6.7U3 or above
+* vSphere CSI/CNS driver 1.0.2 or above
+* Kubernetes 1.14 or above
+
+## Prerequisites
+* Meet the prerequisites of vSphere CSI driver on vSphere roles and privileges - please refer to
+[vSphere Roles and Privileges](https://vsphere-csi-driver.sigs.k8s.io/driver-deployment/prerequisites.html#roles_and_privileges)
+of vSphere CSI Driver.
+* Meet the prerequisites of Virtual Disk Development Kit (VDDK).
+    * Please refer to [Credentials and Privileges for VMDK Access](https://code.vmware.com/docs/11750/virtual-disk-development-kit-programming-guide/GUID-8301C6CF-37C2-42CC-B4C5-BB1DD28F79C9.html)
+of VDDK Programming Guide. **Note: please apply privileges at the vCenter Server level**.
+    * Please make sure port **902** is opened on each ESXi host based on the requirement of the VDDK NBD transport mode.
+
+## Install
+1. [Install Velero](https://velero.io/docs/main/basic-install/)
+2. [Install Object Storage Plugin](#install-object-storage-plugin)
+3. [Install Velero Plugin for vSphere](#install-velero-plugin-for-vsphere)
+
+### Install Object Storage Plugin
+Volume backups are stored in an object store bucket, e.g., S3. Currently these are stored in the same bucket configured for
+the object storage plugin of Velero. Before installing the vSphere plugin, please install and configure the object storage plugin.
+Here is an example, [velero-plugin-for-aws](https://github.com/vmware-tanzu/velero-plugin-for-aws/blob/master/README.md).
+
+
+### Install Velero Plugin for vSphere
+```bash
+velero plugin add <plugin-image>
+```
+
+For Version 1.1.0 the command is
+
+```bash
+velero plugin add vsphereveleroplugin/velero-plugin-for-vsphere:1.1.0
+```
+
+## Uninstall
+To uninstall the plugin, run the following command to remove the InitContainer of velero-plugin-for-vsphere from the Velero deployment first.
+```bash
+velero plugin remove <plugin image>
+```
+To finish the cleanup, delete the Backup Driver deployment, Data Manager daemonset and their related CRDs.
+```bash
+kubectl -n velero delete deployment.apps/backup-driver
+kubectl delete crds backuprepositories.backupdriver.cnsdp.vmware.com \
+                    backuprepositoryclaims.backupdriver.cnsdp.vmware.com \
+                    clonefromsnapshots.backupdriver.cnsdp.vmware.com \
+                    deletesnapshots.backupdriver.cnsdp.vmware.com \
+                    snapshots.backupdriver.cnsdp.vmware.com
+kubectl -n velero delete daemonset.apps/datamgr-for-vsphere-plugin
+kubectl delete crds uploads.datamover.cnsdp.vmware.com downloads.datamover.cnsdp.vmware.com
+```
+
+## Backup
+1. [Backup vSphere CNS Block Volumes](#backup-vsphere-cns-block-volumes)
+2. [Backup vSphere CNS File Volumes](#backup-vsphere-cns-file-volumes)
+### Backup vSphere CNS Block Volumes
+To use the plugin, run a Velero backup and specify the --snapshot-volumes flag.
+
+```bash
+velero backup create my-backup --include-namespaces=my-namespace --snapshot-volumes
+```
+
```

Comment:
Add a link to Velero doc for other backup options, i.e., labels, include/exclude resources, etc.